joomla/joomla-cms
# [4.0] Namespace com_newsfeeds

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_newsfeeds component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_newsfeed.zip )

2. Login to administrator area, access to Components -> Newsfeeds, try to add/edit/publish/unpublish feed item

3. Access to frontend of the site, try to access to some menu items related to newsfeeds, make sure it is working as expected
